runtime/doc/lua.txt

```diff
@@ -1679,8 +1679,8 @@ vim.paste({lines}, {phase})                                      *vim.paste()*
 vim.print({...})                                                 *vim.print()*
     "Pretty prints" the given arguments and returns them unmodified.
 
-    Example: >lua
-        local hl_normal = vim.print(vim.api.nvim_get_hl_by_name('Normal', true))
+    Example:>lua
```

Comment:
The space between `Example:` and `>` disappeared

Reply:
hmmm not sure what happened there, this file was autogenerated...

Reply:
You forgot to regenerate it

Reply:
It seems like removing the blank line (as @zeertzjq pointed out [here](https://github.com/neovim/neovim/pull/25334/files#r1335290416)) also removed the space. It should be restored now.